receiver/m365receiver/factory.go

```diff
@@ -40,7 +40,7 @@ func NewFactory() receiver.Factory {
 func createDefaultConfig() component.Config {
 	return &Config{
 		ScraperControllerSettings: scraperhelper.ScraperControllerSettings{
-			CollectionInterval: 24 * time.Hour,
+			CollectionInterval: 1 * time.Minute,
```

Comment:
Curious why did we move away from the 24 hours? I thought the metrics only updated every 24 hours?

Reply:
that was a mistake by my part, just pushed it with correct interval (changed it to get sample data)